common/tier4_localization_rviz_plugin/src/pose_history/pose_history_display.cpp

```diff
@@ -42,15 +42,12 @@ PoseHistory::PoseHistory() : last_stamp_(0, 0, RCL_ROS_TIME)
   property_line_width_->setMin(0.0);
 }
 
-PoseHistory::~PoseHistory()
-{
-  // Properties are deleted by Qt
-}
+PoseHistory::~PoseHistory() = default;
```

Comment:
Please keep the comment.

Reply:
if keeping the comment, `modernize-use-equals-default` occurs.
Should I ignore it ?

Reply:
And could you tell me what reason to keep the comment?

Reply:
How about writing like this? I've seen code that manually deletes properties. Since QT manages the memory, it will be double freeing. 
```cpp
PoseHistory::~PoseHistory() = default;  // Properties are deleted by Qt
```

Reply:
I got it. I add the comment.